napari/layers/shapes/_shapes_mouse_bindings.py

```diff
@@ -334,15 +568,17 @@ def vertex_remove(layer, event):
     layer.refresh()
 
 
-def _drag_selection_box(layer, coordinates):
+def _drag_selection_box(
+    layer: Shapes, coordinates: Tuple[np.float, ...]
```

Comment:
It is required to have `np.float` not normal `float`?

Reply:
And incidentally, `np.float` *is* `float` and is deprecated:

```python
In [1]: np.float is float
<ipython-input-1-09ec2f1d7cc2>:1: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  np.float is float
Out[1]: True
```

Reply:
resolved with [3fdedf8](https://github.com/napari/napari/pull/5555/commits/3fdedf82c7fe2b36889ad347842451f40178db40) and [3b987c1](https://github.com/napari/napari/pull/5555/commits/3b987c12fd70e0f60b57363ecbb697351411405e)